### PR TITLE
Route all backend server URLs to server binary

### PR DESCRIPTION
**Pull request by mplattu**
Includes also some minor fixes notices while doing this.

### CODE REVIEW

File: server/build/ioris
Comment (lattots) [inline server/build/ioris:1]: Directory is marked in `.gitignore`. Should this be excluded from repo?

File: server/src/html/index.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<html>
```
Comment (lattots) [inline server/src/html/index.html:1]: Missing !DOCTYPE. Don't know if necessary

File: server/build/ioris
Comment (mplattu) [reply server/build/ioris:1]: This commit does exactly what you suggest, it removes the compiled binary from the repository.

Comment (mplattu) [reply server/build/ioris:1]: This commit does exactly what you suggest, it removes the compiled binary from the repository.

File: server/src/html/index.html (hunk 1 above)
Comment (mplattu) [reply server/src/html/index.html:1]: It is necessary, but at the moment the file is just placeholder for the final UI page. Maybe it is better to fix this later when the page itself will be made.

**Pull request merged**